editor/plugins/packed_scene_translation_parser_plugin.cpp

```diff
@@ -50,16 +50,25 @@ Error PackedSceneEditorTranslationParserPlugin::parse_file(const String &p_path,
 	Ref<SceneState> state = Ref<PackedScene>(loaded_res)->get_state();
 
 	Vector<String> parsed_strings;
-	String property_name;
-	Variant property_value;
 	for (int i = 0; i < state->get_node_count(); i++) {
-		if (!ClassDB::is_parent_class(state->get_node_type(i), "Control") && !ClassDB::is_parent_class(state->get_node_type(i), "Viewport")) {
+		String node_type = state->get_node_type(i);
+		if (!ClassDB::is_parent_class(node_type, "Control") && !ClassDB::is_parent_class(node_type, "Viewport")) {
 			continue;
 		}
 
+		String property_name;
+		Variant property_value;
+		Vector<String> new_strings;
+		bool auto_translating = true;
 		for (int j = 0; j < state->get_node_property_count(i); j++) {
 			property_name = state->get_node_property_name(i, j);
-			if (!lookup_properties.has(property_name)) {
+
+			if (property_name == "auto_translate" && (bool)property_value == false) {
+				auto_translating = false;
+				break;
+			}
```

Comment:
`property_value` is assigned further, so it has old value here.

Reply:
Huh? But `property_name` is being assigned at the very beginning of the loop.

Reply:
```
property_name = state->get_node_property_name(i, j); <- name set here

if (property_name == "auto_translate" && (bool)property_value == false) { <- you think it's this property value, but it's previous one
	auto_translating = false;
	break;
}

if (!lookup_properties.has(property_name) || (exception_list.has(node_type) && exception_list[node_type].has(property_name))) {
	continue;
}

property_value = state->get_node_property_value(i, j); <- value set here
```

Reply:
Oops, I brain farted. Thanks for pointing it out.